bin/build

```diff
@@ -57,6 +57,6 @@ else
     cp -r "${MW_LAYER}/dist/." "${DIST_DIR}"
 fi
 
-#set env variable MIDDLEWARE_FUNCTION_URI to the index.js file
-echo -n "$MW_LAYER/dist/index.js" > "$MW_LAYER/env/MIDDLEWARE_FUNCTION_URI"
+mkdir -p "$MW_LAYER/env.launch"
+echo -n "$MW_LAYER" > "$MW_LAYER/env.launch/FUNCTION_URI.override"
```

Comment:
is FUNCTION_URI the system function?
since we "combined" sys+middle, so no more MIDDLEWARE_FUNCTION_URI needed?

Reply:
> is FUNCTION_URI the system function?
> since we "combined" sys+middle, so no more MIDDLEWARE_FUNCTION_URI needed?

That is correct.